lib/discourse_login_client_authenticator.rb

```diff
@@ -1,69 +1,32 @@
 # frozen_string_literal: true
 
 class DiscourseLoginClientAuthenticator < Auth::ManagedAuthenticator
-  class DiscourseLoginClientStrategy < ::OmniAuth::Strategies::OAuth2
-    option :name, "discourse_login"
-
-    option :client_options,
-           authorize_url: "/oauth/authorize",
-           token_url: "/oauth/token",
-           auth_scheme: :basic_auth
-
-    option :authorize_options, [:scope]
```

Comment:
Do we no longer need the `:client_options` and `:authorize_options`?

Reply:
They're all default values, except for "auth_scheme = basic_auth", which I'll restore.